test/suites/apply/apply.go

```diff
@@ -281,7 +294,13 @@ func WaitAllNodeRunningBySSH(s ssh.Interface, masterIP net.IP) {
 		if err != nil {
 			return err
 		}
+		fmt.Println(result)
 		if strings.Contains(result, "NotReady") {
+			resultPod, err := s.CmdToString(masterIP, "kubectl get pod -A", "")
```

Comment:
Using client-go to get pods would better than using kubectl, kubectl would be for sure in official clusterimage, when kubectl's version did not match, this would have an error in test stage. FYI @zhy76

Reply:
Thanks for your great advice and sorry for the late reply, the code in this section uses ssh command to obtain pod information to print logs in the container so that you can see the pod information in the container. At present, simple commands are executed in the container through ssh. Maybe we can use client-go or rpc call etc. to interact with the container in the future if kubectl's version does not match or face other problems.

Reply:
Thanks for your great advice and sorry for the late reply, the code in this section uses ssh command to obtain pod information to print logs in the container so that you can see the pod information in the container. At present, simple commands are executed in the container through ssh. Maybe we can use client-go or rpc call etc. to interact with the container in the future if kubectl's version does not match or face other problems.